include/kokkos-utils/concepts/View.hpp

```diff
@@ -1,15 +1,35 @@
 #ifndef KOKKOS_UTILS_CONCEPTS_VIEW_HPP
 #define KOKKOS_UTILS_CONCEPTS_VIEW_HPP
 
+#include <concepts>
+
 #include "Kokkos_View.hpp"
 
+#include "kokkos-utils/meta/type_traits.hpp"
+
 namespace Kokkos::utils::concepts
 {
 
-//! Concept to specify that a type is a @c Kokkos::View.
+//! Specify that a type is a @c Kokkos::View.
 template <typename T>
 concept View = Kokkos::is_view_v<T>;
 
+//! Specify that a type is a @c Kokkos::View of given rank @p Rank.
+template <typename T, std::size_t Rank>
+concept ViewOfRank = Kokkos::utils::concepts::View<T> && Rank == T::rank;
```

Comment:
```suggestion
concept ViewOfRank = View<T> && T::rank == Rank;
```
(we're already in Kokkos Utils 😄)
(let's make a habit of putting "whatever we want to check" first and the "expected outcome" next)

Reply:
Done for both suggestions :heavy_check_mark: